Dockerfile

```diff
@@ -1,55 +1,24 @@
-FROM ubuntu:18.04
-LABEL maintainer="Mirage"
```

Comment:
Let's change this to `LABEL maintainer="Ghost Proxy"

Reply:
Sounds good, left out as an oversight. Was writing from scratch.